.changes/unreleased/Breaking Changes-20221014-185350.yaml

```diff
@@ -0,0 +1,9 @@
+kind: Breaking Changes
+body: By default incremental materializations will use views if all incremental operations
```

Comment:
Isn't this related to [Upgrade dbt to 1.3](https://github.com/starburstdata/dbt-trino/pull/161/commits/ba64e8f8dd87ca4fd485539858c4df38568d3760) ?

Reply:
I will move it to this one and also squash the incremental refactor into this commit.